notebooks/official/workbench/spark/spark_sample_notebook.ipynb

```diff
@@ -0,0 +1,1200 @@
+{
```

Comment:
<p>Add a link to the compute engine docs on "the region" of the phrase "the region closest to you"</p><p>https://cloud.google.com/compute/docs/regions-zones</p><p> </p><p>Additionally, if there are any regions a user can't use, let them know. For example, if Dataproc is only available in certain regions, specify that they should choose "a region where Dataproc is available" and link to docs showing which ones. This isn't applicable if all products in this notebook are available in all regions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>In terms of the Dataproc cluster, I didn't find any unavailable regions. Please let me know if I'm wrong. Adding a link is done.</p>